content/en/synthetics/multistep.md

```diff
@@ -244,10 +228,16 @@ You can create local variables by clicking **Create Local Variable** at the top
 : Generates an alphanumeric string with `n` characters.
 
 `{{ date(n, format) }}`
-: Generates a date in one of our accepted formats with a value of the date the test is initiated + `n` days.
+: Generates a date in one of Datadog's accepted formats with a value corresponding to the UTC date the test is initiated + or - `n` days.
 
 `{{ timestamp(n, unit) }}` 
-: Generates a timestamp in one of our accepted units with a value of the timestamp the test is initiated at +/- `n` chosen unit.
+: Generates a timestamp in one of Datadog's accepted units with a value corresponding to the UTC timestamp the test is initiated at + or - `n` units.
+
+To obfuscate local variable values in test results, select **Hide and obfuscate variable value**. Once you have defined the variable string, click **Add Variable**.
+
+### Extract variables
```

Comment:
can we keep this section as first section below ### Create local variables
Extracting variables from a step to reuse it in following steps is a key strength of multistep tests